internal/service/advancedclustertpf/model_ClusterDescription20240805.go

```diff
@@ -70,17 +70,26 @@ func NewTFModel(ctx context.Context, input *admin.ClusterDescription20240805, ti
 	}
 }
 
-func NewBiConnectorConfigObjType(ctx context.Context, input *admin.BiConnector, diags *diag.Diagnostics) types.Object {
+func restrictResourceModel(modelOut, modelIn *TFModel) {
```

Comment:
Can we use `overrideAttributesWithPrevStateValue` instead?

Reply:
yes, that's better